pkg/registry/scheduling/rest/storage_scheduling.go

```diff
@@ -79,7 +79,7 @@ func AddSystemPriorityClasses() genericapiserver.PostStartHookFunc {
 	return func(hookContext genericapiserver.PostStartHookContext) error {
 		// Adding system priority classes is important. If they fail to add, many critical system
 		// components may fail and cluster may break.
-		err := wait.Poll(1*time.Second, 30*time.Second, func() (done bool, err error) {
+		err := wait.Poll(1*time.Second, 180*time.Second, func() (done bool, err error) {
```

Comment:
I'm not in favor of increasing timeouts without a more clear understanding of the problem, do we know it can take more than 30 seconds to assign priority classes?

Reply:
Fair.

I was hoping this would fix the issues I have seen.

So the issue is that node_e2e test will go call the apiserver /health endpoint and it returns success.  However, there can be many things that fail.

I notice that the e2e tests for apimachinery actually do a more detailed health check to verify other components are healthy.

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apimachinery/health_handlers.go.

I wonder if this should actually be the change.  Include priority class health checks in the node-e2e tests for verifying that apiserver is healthy.  Currently we just hit /health and if we get a 200 error we assume everything is good.

In this case, I don't really know why it is taking 30 seconds or more.  We basically just want the k8 cluster set up for node-e2e tests and I am seeing that the priority classes bootstrap call is failing in the node-e2e tests.  

I also am unable to get those logs posted to artifacts to actually verify that this is the behavior I am seeing so right now I see that this PR fixed some local issues I see where the priority class bootstrap call fails.  And then we still assume that the apiserver is healthy so everything continues on.

Reply:
ok, let's do this, find just one failure and link it in a comment, so we can go together over the job, past the timestamps and the things you think are relevant

Reply:
So I have https://github.com/kubernetes/kubernetes/issues/121444 to fix an issue where the logs that would show these priority classes issue are not showing up.  

I am hoping to get some advice on this as I notice that jobs that have been migrated to the community cluster seem to be missing logs.  So I tried to add services so I could investigate this more.